scripts/core/CONTRIB.rst

```diff
@@ -1,71 +1,176 @@
+    queue object.
+*   `scripts/core/runtime.yml`_ defines global symbols pertaining to
+    initialization and tear down of the entire runtime.
+*   `scripts/core/registry.yml`_ contains an enumeration of all entry-points,
+    past and present, for use in the XPTI tracing framework. It is
+    automatically updated so shouldn't require manual editing.
+
+.. _scripts/core/common.yml:
+   https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/common.yml
+.. _scripts/core/enqueue.yml:
+   https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/enqueue.yml
+.. _scripts/core/runtime.yml:
+   https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/runtime.yml
+.. _scripts/core/registry.yml:
+   https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/registry.yml
 
-Before your pull request is merged it `must` be reviewed by no less than two 
-code owners of the project `and` be passing all required CI jobs.
+Conformance Testing
+-------------------
 
+For contributions to the core specification conformance tests *should* be
+included as part of your change. The conformance tests can be found
+under ``test/conformance/<component>``, where component refers to the API
+object an entry-point belongs to i.e. platform, enqueue, device.
 
-Conformance Testing
-===================
-
-For contributions to the specification you should include appropriate 
-conformance tests as part of your change. The conformance tests can be found 
-under ``test/conformance/<component>``, where component refers to a set of 
-entry-points i.e. platform, enqueue, device. The conformance tests, at a 
-minimum, should cover at least one test for each of the possible error codes 
-returned, excluding any disaster cases like 
-``${X}_RESULT_ERROR_OUT_OF_HOST_MEMORY`` or similar. Conformance tests should not
-assume anything about the underlying adapter on which they are run against and
-should pass on any conformant adapter.
+The conformance tests *should* ideally include end-to-end testing of all the
+changes to the specification if possible. At minimum, they *must* cover at
+least one test for each of the possible error codes returned, excluding any
+disaster cases like ${X}_RESULT_ERROR_OUT_OF_HOST_MEMORY or similar.
+
+Conformance tests *must* not make assumptions about the adapter under test.
+Tests fixtures or cases *must* query for support of optional features and skip
+testing if unsupported by the adapter.
+
+Experimental Features
+=====================
+
+.. warning::
+
+    Experimental features:
+
+    *   May be replaced, updated, or removed at any time.
+    *   Do not require maintaining API/ABI stability of their own additions
+        over time.
+    *   Do not require conformance testing of their own additions.
+
+Experimental features *must* be defined in two files:
+
+*   ``scripts/core/EXP-FEATURE.rst`` document describing the experimental
```

Comment:
These files do not exist. Is that an omission, or is the first person creating an experimental feature supposed to create them?

Reply:
The intention is for each experimental feature to create a new pair of files describing only that feature. I'll have a go at refining this section to make that clearer.

Reply:
Yes. Thanks!